docs/dev/table/streaming/temporal_tables.md

```diff
@@ -22,9 +22,13 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Temporal Tables represent a concept of a (parameterized) view on a changing history table that returns the content of a table at a specific point in time.
+Temporal tables represent a concept of changing table that its data is deemed to be effective or valid along some time period.
```

Comment:
We don't need to change the definition. It is still valid but we should remove the "history".